pkg/exchange/binance/convert.go

```diff
@@ -218,6 +418,59 @@ func ToGlobalTrade(t binance.TradeV3, isMargin bool) (*types.Trade, error) {
 	}, nil
 }
 
+func ToGlobalFuturesTrade(t futures.AccountTrade) (*types.Trade, error) {
```

Comment:
could be a private function?

Reply:
I didn't get your point. Could you explain why ?

Reply:
Sure, done.